.github/workflows/zephyr.yml

```diff
@@ -63,6 +63,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        zephyr_revision: [
```

Comment:
Please add comments to what is "mnfst" and "zmain".
It is obvious for us, but somebody new to CI yamls may take time to decrypt.